psi4/driver/driver_cbs.py

```diff
@@ -1257,77 +1783,97 @@ def cbs(func, label, **kwargs):
 
         Indicates the sequence of basis sets employed for the second delta correction
         to the correlation energy.
+    
+    .. note::
+        Currently, the basis set set through ``set`` commands have no influence
+        on a cbs calculation.
 
-    * Schemes
-        Transformations of the energy through basis set extrapolation for each
-        stage of the CBS definition. A complaint is generated if number of basis
-        sets in stage_basis does not exactly satisfy requirements of stage_scheme.
-        An exception is the default, ``'xtpl_highest_1'``, which uses the best basis
-        set available. See :ref:`sec:cbs_xtpl` for all available schemes.
+    Schemes
+    -------
+
+    Transformations of the energy through basis set extrapolation for each stage
+    of the CBS definition. A complaint is generated if number of basis sets in 
+    stage_basis does not exactly satisfy requirements of stage_scheme. An exception
+    is the default, ``'xtpl_highest_1'``, which uses the best basis set available.
```

Comment:
Available from where? Best from Psi's list of basis sets? Best from a user-supplied list?

Does best mean "largest zeta" or something else?